packages/service/index.d.ts

```diff
@@ -29,7 +29,7 @@ declare module 'fastify' {
 }
 
 export interface ConfigManagerConfig<T> extends Omit<IConfigManagerOptions, 'source' | 'watch' | 'schema'> {
-  transformConfig: (this: ConfigManager<T>) => Promise<void>
+  transformConfig?: (this: ConfigManager<T>) => Promise<void>
```

Comment:
We can't ship this change. I had to revert it on main because it broke quite a few tests. Morever, this is not correct.